### PR TITLE
Use head/tail over sed for metadata fetching

### PR DESCRIPTION
**Pull request by jlogsdon**
Current sed approach works but slams IO:

```
18543 be/4 root      492.45 K/s    0.00 B/s  0.00 % 99.99 % sed -n 372,372 p /mnt/ad_creatives-20160705204248-74db013351.slug
18842 be/4 root      492.45 K/s    0.00 B/s  0.00 % 99.99 % sed -n 372,372 p /mnt/placement_service-20160704021027-1209abce47.slug
17420 be/4 root      246.23 K/s    0.00 B/s  0.00 % 58.09 % sed -n 372,372 p /mnt/5rocks-server-20160707063520-76904d7940.slug
13609 be/4 root      246.23 K/s    0.00 B/s  0.00 % 55.23 % sed -n 372,372 p /mnt/tapjoydotcom-20160628191805-5ca46ea820.slug
```

That for 20-30 minutes. The head/tail approach is practically instant.

### CODE REVIEW

File: templates/sh.erb [hunk 1]
```diff
@@ -255,7 +255,12 @@ function clean_out_old_releases(){
 function print_package_metadata(){
     local metadata_line=$(grep -a -n -m1 '__METADATA__$' $0 | sed 's/:.*//')
     local archive_line=$(grep -a -n -m1 '__ARCHIVE__$' $0 | sed 's/:.*//')
-    sed -n "$((metadata_line + 1)),$((archive_line - 1)) p" $0
+
+    # This used to be a sed call but it was taking _forever_ and this method is super fast
+    local start_at=$((metadata_line + 1))
+    local take_num=$((archive_line - start_at))
+
+    head -n${start_at} $0 | tail -n${take_num}
```
Comment (jlogsdon) [inline templates/sh.erb]: Instead of eg `sed -n 342,432 p` this is now `head -n342 | tail -n1`.

**Comment by jlogsdon** [comment]
Closing to reopen against master

**Comment by jlogsdon** [comment]
Nevermind, this will also get develop up-to-date.

**Comment by ehealy** [comment]
This LGTM on visual inspection.

**Pull request merged**